src/api/Visualization/types.ts

```diff
@@ -171,6 +172,8 @@ export interface Visualization extends BaseVisualization, PersistedItem {
   user: Pick<User, 'id' | 'username' | 'email'>;
   right: VisualizationRight;
   widgetKey?: string; // defined if the visualization has a widget
+  lastLockedByUser: Pick<User, 'id' | 'username' | 'email'>;
+  locked: boolean;
```

Comment:
@hsenbassam I noticed a couple of things when implementing the backend.

1. The `lastLockedByUser` is not always available, the user can be deleted
    - The correct type is `Pick<User, 'username' | 'email'>`
    - We return "deleted user" when the user is not found
2. The `locked` value is not needed on the Visualization object, only on the visualization tree item
    - We can safely remove it, this simplifies the backend code